bench-run/src/main/scala/dotty/tools/benchmarks/Main.scala

```diff
@@ -58,9 +58,18 @@ object Bench {
   }
 
   def paramsFromFile(file: String): Array[(String, Array[String])] = {
-    Source.fromFile(file).getLines.toArray.map { l =>
```

Comment:
You could use [java.nio.file.Files.readAllLines](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#readAllLines-java.nio.file.Path-):

```scala
Files.readAllLines(Paths.get(file))
  .toArray(new Array[String](0))
  .map { ...
```
Then there is no need to close anything

Reply:
even better, I'll update the PR